src/main/steps/screen-questions/language-preference/content.ts

```diff
@@ -27,15 +27,15 @@ const cy: typeof en = {
   }
 };
 
-export const languagePreferenceForm = {
+export const languagePreferenceForm: FormContent = {
   fields: {
     languagePreferenceWelsh: {
       type: 'radios',
       values: [
         {label: l => l.no, value: 'No'},
         {label: l => l.yes, value: 'Yes'}
       ],
-      validator: value => isFieldFilledIn(value)
+      validator: (value: string): string | void => isFieldFilledIn(value)
```

Comment:
this is unfortunate - it's quite long winded now

Reply:
This might work better if I made it into a reusable `Validator` type.